app/views/catalog/_schema_org.html.erb

```diff
@@ -0,0 +1,5 @@
+<script type="application/ld+json">
+  {
+    "@context": <%= document.itemtype %>,
```

Comment:
Shouldn't the `@context` be `http://schema.org` and the `@type` should be `document.itemtype`

Reply:
Yes most definitely Dropping a WIP on this.